PackagingTest/Carthage/Carthage.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,396 @@
+// !$*UTF8*$!
```

Comment:
Do we need PackagingTest files?

Reply:
I thought it is useful when we will be updating, so that we can test that everything works right away - as part of the release process